assets/apps/dashboard/src/Components/Content/Start.js

```diff
@@ -59,12 +62,12 @@ const Start = (props) => {
 					{!neveDash.isValidLicense && (
 						<p>{neveDash.strings.starterSitesCardUpsellMessage}</p>
 					)}
-					<div
-						className={
-							!neveDash.isValidLicense
-								? 'card-button-group'
-								: 'card-button'
-						}
+
+					<ConditionalWrapper
+						condition={!neveDash.isValidLicense}
+						wrapper={(children) => (
+							<div className="card-button-group">{children}</div>
+						)}
```

Comment:
@cristian-ungureanu I am not sure I understand the need for a conditional wrapper here. I might be mistaken but can't it always be there?